.azure-pipelines/azure-pipelines-build.yml

```diff
@@ -115,6 +115,10 @@ jobs:
             fi
             if [ $(sync_rpc_image) == yes ]; then
               make $BUILD_OPTIONS ENABLE_SYNCD_RPC=y target/docker-syncd-$(platform_rpc)-rpc.gz
+              if [ $PLATFORM_AZP == mellanox ]; then
```

Comment:
i do not quite understand why we are injecting mellanox specific logic in here.

Reply:
@lguohan , in Jenkins, we have a separate job with RPC enabled for mellanox only. Prefer to use the same job for the same platform. For one platform, we can have multiple images by design. Mellanox still needs the image with rpc enabled and the one with rpc none-enabled.

Reply:
See https://sonic-jenkins.westus2.cloudapp.azure.com/job/mellanox/job/buildimage-mlnx-202012-rpc/ as an example.

Reply:
then introduce a new variable to build rpc image

Reply:
sync_rpc_image is a little misleading, rename to syncd_rpc_image. Use the new parameter syncd_rpc_image to build the bin image.